library/std/src/sys/pal/unix/process/process_vxworks.rs

```diff
@@ -68,7 +67,7 @@ impl Command {
                 .as_ref()
                 .map(|c| c.as_ptr())
                 .unwrap_or_else(|| *sys::os::environ() as *const _);
-            let stack_size = thread::min_stack();
+            let stack_size = libc::PTHREAD_STACK_MIN;
```

Comment:
As a note, I am assuming this is at least 64KiB on VxWorks?

Reply:
The previous sources had a min_stack() function that varied the stack size as specified by the user. Why was that removed?

Reply:
note: we would appreciate a report at https://github.com/rust-lang/rust
help: you can increase rustc's stack size by setting RUST_MIN_STACK=33554432
note: backtrace dumped due to SIGSEGV! resuming signal
/folk/prj-diab-dev/deepa/vx2403/test/vsb/usr/rust/../../usr/rust/rustc.linux: line 5: 126255 Segmentation fault      $WIND_HOME/compilers/rust-1.79.0.0/x86_64-unknown-linux-gnu/bin/rustc --sysroot=$WIND_HOME/compilers/rust-1.79.0.0/x86_64-unknown-linux-gnu/bin/../ "$@"
error: could not compile `remove` (bin "remove")

Caused by:
  process didn't exit successfully: `/folk/prj-diab-dev/deepa/vx2403/test/vsb/usr/rust/../../usr/rust/rustc.linux --crate-name remove --edition=2021 src/main.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat --diagnostic-width=145 --crate-type bin --emit=dep-info,link -C embed-bitcode=no -C debuginfo=2 -C metadata=471eb2f67ad5bc6b -C extra-filename=-471eb2f67ad5bc6b --out-dir /folk/prj-diab-dev/deepa/vx2403/test/vsb/usr/rust/remove/target/i686-wrs-vxworks/debug/deps --target i686-wrs-vxworks -C incremental=/folk/prj-diab-dev/deepa/vx2403/test/vsb/usr/rust/remove/target/i686-wrs-vxworks/debug/incremental -L dependency=/folk/prj-diab-dev/deepa/vx2403/test/vsb/usr/rust/remove/target/i686-wrs-vxworks/debug/deps -L dependency=/folk/prj-diab-dev/deepa/vx2403/test/vsb/usr/rust/remove/target/debug/deps` (exit status: 139)
  Now that I checked this, I get segmentation fault when I compile a program with cargo. This has to be changed.

Reply:
I'm getting a segmenation fault when I compile programs with cargo. Can this be causing that? But I'm able to compile programs using rustc.

Reply:
Possibly? I recommend that this starts much larger, around 64KiB. 4096B is much too small, even for tiny embedded systems. We can correct it in a later change if it needs to be corrected.

Reply:
But when I compile a program for vxworks, I'm running the compiler on Linux. So changes in process_vxworks.rs should not have caused this.

Reply:
It was 256kb till rust 1.78. I'll set the default to 256kb.

Reply:
Oh, then no, if everything is a Linux host then it shouldn't matter.